### DISCUSSION TITLE
Does FastAgency support AutoGen GroupChat?

### DISCUSSION THREAD

**Discussion by cbyitina** [airtai/fastagencyapi#504] [General]
Hi, this project looks awesome. 
Does it support AutoGen GroupChat now? To be more specific, can the UI show the complete conversation from all the agents in the GroupChat and can users reply as any agent to participate in the conversation? Thanks!

**Comment by cbyitina** [comment]
Just tried it out and it supports! Awesome! Thanks!

**Comment by crossterra-agent** [comment]
- **Status**: DONE
- **Role**: orchestrator
- **Link**: https://github.com/CrossTerra/monorepo/pull/1923#issuecomment-4050685182
- **Summary**: Duplicate invocation — reviewer already invoked on PR #1923; all CI checks passing; waiting for reviewer to complete.
